pkg/agent/flowexporter/connections/conntrack_ovs.go

```diff
@@ -246,6 +248,27 @@ func flowStringToAntreaConnection(flow string, zoneFilter uint16) (*flowexporter
 				return nil, fmt.Errorf("conversion of timeout %s to int failed", fields[len(fields)-1])
 			}
 			conn.Timeout = uint32(val)
+		case strings.Contains(fs, "labels"):
+			fields := strings.Split(fs, "=")
+			labelStr := strings.Replace(fields[len(fields)-1], "0x", "", -1)
+			if len(labelStr) < 16 {
+				labelStr = strings.Repeat("0", 16-len(labelStr)) + labelStr
+			}
+			egressStr, ingressStr := labelStr[:8], labelStr[8:16]
+			egressVal, err := hex.DecodeString(egressStr)
+			if err != nil {
+				return nil, fmt.Errorf("conversion of egressStr %s to []byte failed", egressStr)
+			}
+			ingressVal, err := hex.DecodeString(ingressStr)
+			if err != nil {
+				return nil, fmt.Errorf("conversion of ingressStr %s to []byte failed", ingressStr)
+			}
+			egressOfID := binary.BigEndian.Uint32(egressVal)
+			ingressOfID := binary.BigEndian.Uint32(ingressVal)
+			buf := make([]byte, 8)
+			binary.LittleEndian.PutUint32(buf[:4], ingressOfID)
+			binary.LittleEndian.PutUint32(buf[4:], egressOfID)
```

Comment:
As the string is already decoded into byte slice, we could just change the endianness directly instead of doing decoding and encoding of individual uint32 vlaues. Reversing the byte slice can work:

```
hexval, _ := hex.DecodeString(str)
for i:= 0; i < len(hexval)/2; i++ {
     hexval[i], hexval[len(hexval)-i-1] = hexval[len(hexval)-i-1], hexval[i]
}
```

Reply:
Thanks, addressed.